ztoc/ztoc_test.go

```diff
@@ -391,15 +440,20 @@ func TestZtocGeneration(t *testing.T) {
 }
 
 func TestZtocSerialization(t *testing.T) {
+	for _, tc := range testZtocs {
+		testZtocSerialization(t, tc.compressionAlgo, tc.tarGenerator)
+	}
+}
+
+func testZtocSerialization(t *testing.T, compressionAlgo string, generator tarGenerator) {
```

Comment:
I think this naming scheme may be confusing to read later.

Reply:
I think it's common to have a short exported func for preprocessing/sanity check/etc and put detailed logic in a unexported func with the same name. We have similar tests in other test cases (e.g. [here](https://github.com/awslabs/soci-snapshotter/blob/fc45d633a63e4d8512ac2e4b264bd3a19e5a9d47/fs/reader/reader_test.go#L66)).

Any suggestion to improve the readability or reduce confusion while able to test all various ztoc compressions?